node.gyp

```diff
@@ -606,25 +613,25 @@
       ],
 
       'libraries': [
-        '<(OBJ_GEN_PATH)/node_javascript.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/node_debug_options.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/async-wrap.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/env.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/node.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/node_buffer.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/node_i18n.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/node_url.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/debug-agent.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/util.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/string_bytes.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/string_search.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/stream_base.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/node_constants.<(OBJ_SUFFIX)',
-        '<(OBJ_PATH)/node_revert.<(OBJ_SUFFIX)',
-        '<(OBJ_TRACING_PATH)/agent.<(OBJ_SUFFIX)',
-        '<(OBJ_TRACING_PATH)/node_trace_buffer.<(OBJ_SUFFIX)',
-        '<(OBJ_TRACING_PATH)/node_trace_writer.<(OBJ_SUFFIX)',
-        '<(OBJ_TRACING_PATH)/trace_event.<(OBJ_SUFFIX)',
+        '<(OBJ_GEN_PATH)<(OBJ_SEPARATOR)node_javascript.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)node_debug_options.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)async-wrap.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)env.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)node.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)node_buffer.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)node_i18n.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)node_url.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)debug-agent.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)util.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)string_bytes.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)string_search.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)stream_base.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)node_constants.<(OBJ_SUFFIX)',
+        '<(OBJ_PATH)<(OBJ_SEPARATOR)node_revert.<(OBJ_SUFFIX)',
+        '<(OBJ_TRACING_PATH)<(OBJ_SEPARATOR)agent.<(OBJ_SUFFIX)',
+        '<(OBJ_TRACING_PATH)<(OBJ_SEPARATOR)node_trace_buffer.<(OBJ_SUFFIX)',
+        '<(OBJ_TRACING_PATH)<(OBJ_SEPARATOR)node_trace_writer.<(OBJ_SUFFIX)',
+        '<(OBJ_TRACING_PATH)<(OBJ_SEPARATOR)trace_event.<(OBJ_SUFFIX)',
```

Comment:
Still it probably should be `'OBJ_SEPARATOR': '/node.'`, with no trailing slashes in other variables.